app/controllers/api/v1/users_controller.rb

```diff
@@ -1,9 +1,8 @@
 module Api
   module V1
     class UsersController < V1::BaseController
-      include Foreman::Controller::UsersMixin
-
       before_filter :find_resource, :only => %w{show update destroy}
+      include Foreman::Controller::UsersMixin
```

Comment:
ah now I get it, it's actual fix, as the mixin adds before filters, could you add # comment so we don't reorder it later?